hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -124,12 +123,12 @@ public class AsyncConnectionImpl implements AsyncConnection {
   private final ClusterStatusListener clusterStatusListener;
 
   public AsyncConnectionImpl(Configuration conf, ConnectionRegistry registry, String clusterId,
-    SocketAddress localAddress, User user) {
-    this(conf, registry, clusterId, localAddress, user, Collections.emptyMap());
+    User user) {
+    this(conf, registry, clusterId, user, Collections.emptyMap());
```

Comment:
The diff across commits is sort of confusing because this class is different across branch-2 and master. [The branch-2 version](https://github.com/apache/hbase/blob/branch-2/hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java#L123-L124) does not have a SocketAddress argument. [The master version](https://github.com/apache/hbase/blob/master/hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java#L130-L131) does. So I think this is the correct changeset for the branch-2 backport

Reply:
whoops, you're right. i was tricked by trying to just look at the specific commits.